rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -112,17 +118,78 @@ def run_check(cls, results_dir, **kwargs):
             command.append(pkg)
             command.append(new_pkg)
             logger.debug('Package name for ABI comparison %s', old_name)
-            output = os.path.join(cls.results_dir, results_dir, old_name + '-' + cls.log_name)
+            output = os.path.join(cls.results_dir, old_name + '.txt')
             try:
                 ret_code = ProcessHelper.run_subprocess(command, output_file=output)
             except OSError:
                 raise CheckerNotFoundError("Checker '%s' was not found or installed." % cls.CMD)
 
             if int(ret_code) & settings.ABIDIFF_ERROR and int(ret_code) & settings.ABIDIFF_USAGE_ERROR:
                 raise RebaseHelperError('Execution of %s failed.\nCommand line is: %s' % (cls.CMD, cmd))
-            if int(ret_code) == 0:
-                text = 'ABI of the compared binaries in package %s are equal.' % old_name
-            else:
-                text = 'ABI of the compared binaries in package %s are not equal.' % old_name
-            reports[output] = text
-        return reports
+            ret_code = int(ret_code)
+            reports[old_name] = ret_code
+        return {'packages': cls.parse_abi_logs(reports), 'abi_changes': cls.abi_changes, 'path': cls.results_dir}
+
+    @classmethod
+    def parse_abi_logs(cls, reports):
+        """
+        Parse abipkgdiff logs
+        :param reports: dictionary with paths to the logs of the produced (sub)packages
+        :return: returns packages names with its abipkgdiff changes
+        """
+        def parse_changes(chgs):
+            """
+            Parses each line with abipkgdiff output info
+            :param chgs: lines of changes
+            :return: dictionary of list of changes
+            """
+            ret_changes = {}
+            for line in chgs:
+                # Find all numbers of added/changed/deleted and get rid of '0' changes
+                match = re.findall(r'\d+ \w+', line)
```

Comment:
This is too generic, you should search for specific patterns (*Added*, *Removed*, *Changed*). Take this for example:
```
Functions changes summary: 0 Removed, 2 Changed (144 filtered out), 2 Added functions
Variables changes summary: 0 Removed, 0 Changed (1 filtered out), 0 Added variable
```

You have to consider also filtered-out symbols.